aws/resource_aws_lb_target_group_test.go

```diff
@@ -794,6 +794,39 @@ resource "aws_vpc" "test" {
 }`, targetGroupName)
 }
 
+func testAccAWSLBTargetGroupConfig_typeTCP(targetGroupName string) string {
```

Comment:
This configuration isn't actually used in a test, was the corresponding test omitted?